docs/eck-attributes.asciidoc

```diff
@@ -1,4 +1,4 @@
-:eck_version: 2.11.0
+:eck_version: 2.11.1
 :eck_crd_version: v1
 :eck_release_branch: 2.11
```

Comment:
I double checked some old tags (2.6.1 for example) to ensure this release branch stayed the same..